newsfragments/99.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+:func:`asyncio.get_running_loop()` will now return the trio-asyncio event loop
```

Comment:
I believe `:func:` wants its referent to be specified without the trailing parens. This made me realize we aren't testing newsfragments as part of the docs build. I created #102 to fix.

Reply:
Never mind, it appears to work either way!